src/modules/navigator/rtl_params.c

```diff
@@ -44,9 +44,10 @@
  */
 
 /**
- * RTL altitude
+ * Return to home relative altitude.
  *
- * Altitude to fly back in RTL in meters
+ * Relative altitude above home at which the vehicle will return during RTL mode.
```

Comment:
Perhaps "Altitude above home at which the vehicle will return during return mode."

1. Altitude above home is by definition relative.
2. We're using Return mode in QGC. RTL is of course embedded in the psych!